config.ini

```diff
@@ -0,0 +1,19 @@
+[Default]
+PORT: 8084
+HOST: 127.0.0.1
+START_CMD: python server.py
```

Comment:
Might be worth putting a comment next to this to review it before we use it.  The others have a specific version (eg, python3.4 server.py) and I don't know why this one doesnt....  It should probably be consistant

Reply:
This was due to gitlab config. For some reason this one had a different start command to any the other repo's